client/signup/steps/theme-selection/index.jsx

```diff
@@ -66,13 +66,34 @@ class ThemeSelectionStep extends Component {
 	};
 
 	renderThemesList() {
+		const pressableWrapperClassName = classNames( {
+			'theme-selection__pressable-wrapper': true,
```

Comment:
If this key is always true I think classnames allows single strings as arguments. Also, check which is more common in calypso: classnames or classNames